### PR TITLE
feat(compute-profiles): Change profile settings according to instance types in RaBe Standard-VM definition

### PR DESCRIPTION
**Pull request by smirta**

### CODE REVIEW

File: roles/infrastructure/tasks/compute_profiles.yaml [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+    name: theforeman.foreman.compute_profiles
+  vars:
+    foreman_compute_profiles:
+      - name: 1-Small
+        compute_attributes:
+          - compute_resource: server-008.dmz-admin.int.rabe.ch
+            vm_attrs:
+              cpus: 1
+              memory: 2147483648
+              image_id: /var/lib/libvirt/images/AlmaLinux-9-GenericCloud-latest.x86_64.qcow2
```
Comment (smirta) [inline roles/infrastructure/tasks/compute_profiles.yaml:13]:
```suggestion
              image_id: /var/lib/libvirt/images/AlmaLinux-9-GenericCloud-latest.x86_64.qcow2
              autostart: true
```
we need to test this as soon as we have added this PR to the main branch and everything is working as expected. https://github.com/fog/fog-libvirt supports this and it seems as if Foreman has these parameters in the backend
```
# hammer compute-profile info --id 1
[...]
    VM attributes:    {"cluster"=>"ee26f4bd-0f5b-4b9b-9074-7ca55db80ffc", "template"=>"c7886e3d-7b3e-4fc5-8b92-ce34e87a20d7", "instance_type"=>"00000007-0007-0007-0007-00000000010a", "memory"=>"4294967296", "ha"=>"0", "display"=>{"type"=>"spice", "keyboard_layout"=>"en-us"}, 
[...]
```

Comment (smirta) [reply roles/infrastructure/tasks/compute_profiles.yaml:13]: wdyt @hairmare context: autostart of VMs

Comment (hairmare) [reply roles/infrastructure/tasks/compute_profiles.yaml:13]: I *think* that this should be passed through to fog. we'll have to test if that holds true, maybe it needs to be on the vm and not the template, or maybe fam needs patching to support it all the way

**Comment by hairmare** [comment]
Do we need to further align this with our internal wiki docs before merging it?

Maybe we can move the actual sizing to vars and use the same sizes across all virt platforms (incl. ovirt).

File: roles/infrastructure/tasks/compute_profiles.yaml (hunk 1 above)
Comment (hairmare) [reply roles/infrastructure/tasks/compute_profiles.yaml]: our current profiles seem to be loading 20240507,  but we specify latest... maybe this doesn't work?

**Comment by hairmare** [comment]
closing for now because of: https://github.com/radiorabe/ansible-collection-rabe_foreman/pull/173

feel free to re-open, i'm not deleting the branch

**Pull request closed without merging**